### DISCUSSION TITLE
userguide.html is no longer in /usr/share/doc/i3/

### DISCUSSION THREAD

**Discussion by jdjohnston** [i3/i3#6083] [Q&A] [answered]
This _might_ be an Arch packaging issue, but I  was disappointed when I  found that `/usr/share/doc/i3/` no longer included `userguide.html `.  Not everyone has an always-on 'Net connection

**Comment by orestisfl** [comment, marked as answer]
Report that to the arch package maintainers. The `i3-git` package does place the files in `/usr/share/doc/i3/` and the files are included in the latest release from https://i3wm.org/downloads/.

**Reply by jdjohnston** [reply]
I think I'll switch to `i3-git`. Thanks for the pointer.
